rellax.js

```diff
@@ -278,19 +271,27 @@
         var translate = 'translate3d(' + (self.options.horizontal ? positionX : '0') + 'px,' + (self.options.vertical ? positionY : '0') + 'px,' + zindex + 'px) ' + blocks[i].transform;
         self.elems[i].style[transformProp] = translate;
       }
-      self.options.callback(positions);
+      self.options.callback();
```

Comment:
Curious why you removed the positions argument? I didn't write this code, but it seems like we pass that back as a return value on callback?

Reply:
Oh yeah, sorry I didn’t mention that. It failed JSHint as `positions` is out of scope, and as I failed to notice the reference to that feature in the docs, I thought it was an error/leftover code. Still I’m not sure it works properly as `positions` is set in a loop?